doc/examples/particles/lib/main.dart

```diff
@@ -497,15 +500,18 @@ class MyGame extends BaseGame {
     super.render(canvas);
 
     if (debugMode()) {
-      fpsTextConfig.render(canvas, '${fps(120).toStringAsFixed(2)}fps',
-          Position(0, size.height - 24));
+      fpsTextConfig.render(
+          canvas, '${fps(120).toStringAsFixed(2)}fps', Vector2(0, size.y - 24));
     }
   }
 
   /// Returns random [Offset] within a virtual
   /// grid cell
   Offset randomCellOffset() {
-    return cellSize.scale(rnd.nextDouble(), rnd.nextDouble()) - halfCellSize;
+    return Offset(
+      cellSize.x * rnd.nextDouble() - halfCellSize.x,
+      cellSize.y * rnd.nextDouble() - halfCellSize.y,
+    );
```

Comment:
`return cellSize * Vector2(rnd.nextDouble(), rnd.nextDouble()) - halfCellSize` :D

Reply:
That one is still an Offset since it uses the const constructor, we could turn it into Vector2. But there might be some performance loss, what do you think?